packages/lavamoat-node/test/globals.spec.js

```diff
@@ -19,6 +19,47 @@ test('globals - has only the expected global circular refs', async (t) => {
   scenario.checkResult(t, testResult.sort(), scenario)
 })
 
+test('globals - circular refs taming', async (t) => {
```

Comment:
(suggestion) add a similar test for `@lavamoat/node`

though I am not entirely sure why we can't add a single scenario in `core`, since `lavamoat-node`, `lavamoat-browserify` and `@lavamoat/node` run all of the core scenarios?

Reply:
that was the case originally, then i realized each env has its own set of globalThis aliases, and we want to test each separately, partly to tell passing config from each of them actually works

Reply:
ah, I see. there wouldn't be any elegant way to do this, then.

hopefully I can address that one of these days